### ISSUE TITLE
A couple of tools i think should be mentioned unless there a reasons they shouldnt that are unknown to me..

### ISSUE THREAD

**Issue by Lctsolutions** [hahwul/MobileHackersWeapons#5]
Andrax, pentesting platform similar to Nethunter
PCAPdroid
Termux(maybe obvious but its been extremely helpful to me in these endeavors)
HiJack suite
Hijacker
Droidsheep(tho this is old now) - a remake of firesheep
PortAuthority/Portdroid
Netcut
BeHe keyboard or hackers keyboard
JsHook
Metasploit(mentioned already but there are other builds for mobile devices)
Magisk

**Comment by hahwul** [comment]
@Lctsolutions 
Oh, Thank you for your opinion! Can you send it to Pull Request? Or should I?
(I thought it would be better to be added to the contributor via PR😊)

> [CONTRIBUTING.md](https://github.com/hahwul/MobileHackersWeapons/blob/main/CONTRIBUTING.md)

**Comment by Lctsolutions** [comment]
Yes please do a pull for me. Im still learning my way around github and cant seem to get it working
